apps/transport/lib/transport_web/live/backoffice/irve_dashboard_live.ex

```diff
@@ -0,0 +1,125 @@
+defmodule TransportWeb.Backoffice.IRVEDashboardLive do
+  use Phoenix.LiveView
+  use Phoenix.HTML
+  import TransportWeb.Backoffice.JobsLive, only: [ensure_admin_auth_or_redirect: 3]
+  import TransportWeb.Router.Helpers
+
+  @impl true
+  def mount(_params, %{"current_user" => current_user} = _session, socket) do
+    {:ok,
+     ensure_admin_auth_or_redirect(socket, current_user, fn socket ->
+       :ok = Oban.Notifier.listen([:gossip])
+
+       socket
+       |> assign_data()
+     end)}
+  end
+
+  @doc """
+  A quick and dirty filter on some interesting fields, using downcase matching.
+  """
+  def must_list_resource?(resource, filter) do
+    filter = filter |> String.downcase() |> String.trim()
+
+    filter == "" ||
+      String.contains?(resource["dataset_organisation_name"] |> String.downcase(), filter) ||
+      String.contains?(format_validity(resource["valid"]) |> inspect |> String.downcase(), filter)
```

Comment:
```suggestion
      String.contains?(format_validity(resource["valid"]) |> String.downcase(), filter)
```
Ah il y a un filtre pour pouvoir chercher par OK ou KO

Reply:
Yes, c'est un peu bricolé au scotch mais ça fait le taf pour ce tour :smile: